lib/webidl2.js

```diff
@@ -6,7 +6,7 @@
         ,   re = {
                 "float":        /^-?(([0-9]+\.[0-9]*|[0-9]*\.[0-9]+)([Ee][-+]?[0-9]+)?|[0-9]+[Ee][-+]?[0-9]+)/
             ,   "integer":      /^-?(0([Xx][0-9A-Fa-f]+|[0-7]*)|[1-9][0-9]*)/
-            ,   "identifier":   /^[A-Z_a-z][0-9A-Z_a-z]*/
+            ,   "identifier":   /^[A-Z_a-z][0-9A-Z_a-z-]*/
```

Comment:
@heycam, is this actually a good idea in [WebIDL](https://heycam.github.io/webidl/#idl-grammar)? It'll make identifiers that can't be used in Javascript. :-/